src/test/java/net/imglib2/algorithm/bspline/GenBsplineKernels.java

```diff
@@ -0,0 +1,525 @@
+package net.imglib2.algorithm.bspline;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import net.imglib2.algorithm.interpolation.randomaccess.BSplineInterpolator;
+
+public class GenBsplineKernels {
+
+	public static void main(String[] args) throws IOException
```

Comment:
What is this?

Reply:
old test. will remove

Reply:
err, some of that is old tests. Other parts I want to keep - namely they generate the numbers that go [here](https://github.com/bogovicj/imglib2-algorithm/blob/bsplineInterpolation/src/main/java/net/imglib2/algorithm/bspline/BsplineKernel4.java#L18-L26), for example.  I'd want to keep that around somewhere

Reply:
documented this class better.  Tests as junit now in their own class.